rel-eng/tito.props

```diff
@@ -398,9 +404,10 @@ whitelist = gyp
   nodejs-jquery
   nodejs-js-tokens
   nodejs-lodash
-  nodejs-lodash.debounce
   nodejs-lodash-es
+  nodejs-lodash.debounce
```

Comment:
For historical reasons we sort with en_GB:
https://github.com/theforeman/foreman-packaging/blob/763e83c87d9a6d0f73177163dc2ea47298ed1506/add_gem_package.sh#L52-L61

Please keep it sorted consistently so we don't go back and forth when the script is ran again.